tests/Aspire.Playground.Tests/Aspire.Playground.Tests.csproj

```diff
@@ -4,13 +4,13 @@
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
     <!-- Set this explicitly so the project can build without arcade -->
     <IsTestProject>true</IsTestProject>
-    <ExcludeTestUtilities>true</ExcludeTestUtilities>
+    <IncludeTestUtilities>false</IncludeTestUtilities>
 
-    <!-- no docker support on helix/windows yet -->
-    <RunTestsOnHelix Condition="'$(OS)' != 'Windows_NT'">true</RunTestsOnHelix>
```

Comment:
I added `RunsOnHelixWindows` and `RunsOnHelixLinux` props because of this.

I understand that these two props have no direct impact (because Helix payload is bootstrapped in its own way), but they make it much easier to reason about each test project capabilities.